keylime/keylime_agent.py

```diff
@@ -114,14 +119,20 @@ def do_GET(self):
 
             # Sanitization assurance (for tpm.run() tasks below)
             if not (nonce.isalnum() and
-                    (pcrmask is None or pcrmask.isalnum()) and
-                    (vpcrmask is None or vpcrmask.isalnum() and
-                    ima_ml_entry.isalnum())):
+                    (pcrmask is None or config.valid_hex(pcrmask)) and
```

Comment:
why drop the check for `vpcrmask`?

Reply:
Because we don't have vtpm support in the Python agent. I think it does not make sense to validate data that we do not use.